src/hci/gnomex/controller/ChangePassword.java

```diff
@@ -203,34 +203,34 @@ else if (appUser.getIsActive() != null && appUser.getIsActive().equals("N")) {
       this.validate();
     } catch (HibernateException e) {
       e.printStackTrace();
-      log.error(e.getClass().toString() + ": " + e);
+      log.error(e.getClass().toString() + ": ", e);
```

Comment:
I may be butting in here, but I think you could consolidate all five of these catch blocks into one multi-catch.  Let me know, If you are not familiar with the syntax.